package.json

```diff
@@ -21,22 +22,25 @@
     "test:server": "ember test --server"
   },
   "devDependencies": {
-    "@types/qunit": "^2.5.3",
-    "broccoli-funnel": "^2.0.1",
-    "broccoli-merge-trees": "^3.0.1",
-    "broccoli-rollup": "^2.1.1",
-    "broccoli-typescript-compiler": "^4.0.1",
+    "@types/qunit": "^2.11.1",
+    "broccoli-funnel": "^3.0.4",
+    "broccoli-merge-trees": "^4.2.0",
+    "broccoli-rollup": "^4.1.1",
+    "broccoli-typescript-compiler": "^7.0.0",
     "do-you-even-bench": "^1.0.5",
-    "ember-cli": "^3.4.2",
-    "ember-cli-dependency-checker": "^3.0.0",
-    "ember-cli-inject-live-reload": "1.7.0",
-    "glob": "^7.1.3",
+    "ember-cli": "^3.26.1",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-inject-live-reload": "2.0.2",
+    "glob": "^7.1.6",
     "loader.js": "^4.7.0",
-    "lolex": "^2.7.5",
-    "qunit": "^2.6.2",
-    "rollup-plugin-buble": "^0.19.2",
-    "rollup-plugin-commonjs": "^9.1.8",
-    "rollup-plugin-node-resolve": "^3.4.0",
-    "tslint": "^5.11.0"
+    "lolex": "^6.0.0",
+    "qunit": "^2.14.1",
+    "rollup-plugin-buble": "^0.19.8",
+    "rollup-plugin-commonjs": "^10.1.0",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "tslint": "^6.1.3"
+  },
+  "engines": {
+    "node": "10.* || >= 12.*"
```

Comment:
Do we need this? Adding it is basically a breaking change, but none of the other changes seem breaking...

Reply:
I was having trouble running things locally without it, but I can try again.

Reply:
Looks like it was unnecessary. I removed it.